audit/projects/k8s-staging-boskos/services/logging/logs.json

```diff
@@ -1,4 +1,3 @@
 [
-  "projects/k8s-staging-boskos/logs/cloudaudit.googleapis.com%2Factivity",
-  "projects/k8s-staging-boskos/logs/cloudbuild"
+  "projects/k8s-staging-boskos/logs/cloudaudit.googleapis.com%2Factivity"
```

Comment:
Noise in logging export: https://github.com/kubernetes/k8s.io/issues/2111

This is an example of cloudbuild logs aging out